kernelArgs.go

```diff
@@ -0,0 +1,57 @@
+// Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
nit: I think it's a bit odd to have `kernelArgs.go` as the file name.  I'd vote for either `kernelargs.go`, `kernel.go`, or `kernel_args.go`.

Reply:
Sure, I have no preference either way